tests/integration/crossref/test_crossref_views.py

```diff
@@ -85,7 +85,7 @@ def test_crossref_search_handles_the_response_when_the_request_is_valid(log_in_a
 def test_crossref_search_handles_the_response_when_the_request_asks_for_a_non_existing_doi(log_in_as_scientist, app_client):
     with requests_mock.Mocker() as requests_mocker:
         requests_mocker.register_uri(
-            'GET', 'http://api.crossref.org/works/does-not-exist',
+            'GET', 'http://api.crossref.org/works/None',
```

Comment:
This suspiciously looks like it's stringifying a `None` python object, are you sure that's correct? (maybe it should not be doing the request at all if the object is `None`).

Reply:
In that case I need to remove this test?

Reply:
I'd say no, better avoid doing that request at all, like just removing the `requests_mocker` part, and making sure it does not do the http request (as doing a request  like that will surely not work, we skip the whole external request at all).